packages/clerk-js/src/ui/new/renderer.tsx

```diff
@@ -1,10 +1,9 @@
-// TODO: don't import here
-import '@clerk/ui/styles.css';
-
 import { ClerkInstanceContext, OptionsContext } from '@clerk/shared/react';
 import type { ClerkHostRouter } from '@clerk/shared/router';
 import { ClerkHostRouterContext } from '@clerk/shared/router';
 import type { ClerkOptions, LoadedClerk } from '@clerk/types';
+//@ts-ignore - This is treated as a string export by Webpack
```

Comment:
[question] Would it be better to use`@ts-expect-error` so in the case this is no longer an issue in the future the `@ts-ignore` doesn't just stick around forever?